### PR TITLE
Add weighted mean docs.

### PR DESCRIPTION
**Pull request by dcherian**
I like @fujiisoup's weighted mean demo in this stack overflow example:
https://stackoverflow.com/questions/48510784/xarray-rolling-mean-with-weights

I thought it'd be a useful addition to the docs on rolling.

### CODE REVIEW

File: doc/computation.rst [hunk 1]
```diff
@@ -185,6 +185,12 @@ windowed rolling, convolution, short-time FFT, etc.
 
 Because the ``DataArray`` given by ``r.construct('window_dim')`` is a view
 of the original array, it is memory efficient.
+One can also use ``construct`` to compute a weighted mean:
```
Comment (maxim-lian) [inline doc/computation.rst]: `weighted rolling`
And nit: we've used `You` or `We` throughout - let's not add another voice

File: doc/computation.rst [hunk 2]
```diff
@@ -185,7 +185,7 @@ windowed rolling, convolution, short-time FFT etc.
 
 Because the ``DataArray`` given by ``r.construct('window_dim')`` is a view
 of the original array, it is memory efficient.
-One can also use ``construct`` to compute a weighted mean:
+You can also use ``construct`` to compute a weighted mean:
```
Comment (maxim-lian) [inline doc/computation.rst]: I do think this should explicitly say `weighted rolling mean`, may have got lost from https://github.com/pydata/xarray/pull/2012#discussion_r176850037

File: doc/computation.rst (hunk 1 above)
Comment (dcherian) [reply doc/computation.rst]: Ah, I missed that

**Comment by maxim-lian** [comment]
Thanks a lot @dcherian ! 
And @fujiisoup for the feature and SO post!

**Pull request merged**